.github/deploy-keys.sh

```diff
@@ -61,3 +61,7 @@ deploy_env "${THIS}-dist"
 #   - npm-install.yml
 #   - release.yml
 deploy_env node-cache
+
+# https://github.com/cockpit-project/org.cockpit_project.CockpitClient
+#   - update-flathub.yml
+deploy_env flathub org.cockpit_project.CockpitClient
```

Comment:
Can we have a deploy key for https://github.com/flathub/org.cockpit_project.CockpitClient ? I suppose that still won't help us with creating a PR (at least not with the default github token), but it would at least avoid having to keep this fork on cockpit-project -- we could just directly create a branch on flathub/org.cockpit_project.CockpitClient and then propose that.

Reply:
The short answer is "no".  But I can ask.

Reply:
Ack, thanks. It would alleviate the need for keeping a fork, but it's not a big deal. I'd just like to sort out the syncing, see above.